WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -750,10 +750,14 @@ local function modify(parent, region, data)
     -- Positioning is based on grow information from the data
     if not self:IsSuspended() then
       self.needToPosition = false
-      if animate then
-        WeakAuras.RegisterGroupForPositioning(data.id, self)
+      if #self.activeRegions > 0 then
```

Comment:
I think this meant to be self.controlledChildren

Reply:
with `if #self.sortedChildren > 0 then`, as discussed on discord, it fix the issue

Reply:
```suggestion
      if #self.sortedChildren > 0 then
```